[DOCS] Remove all mentions of profilers in public docs (#8825)
great-expectations/great_expectations: docs/docusaurus/docs/terms/batch_request.md

## Relationship to other objects

> A Batch Request is always used when Great Expectations builds a Batch. Any time you interact with something that requires a Batch of Data (such as a <TechnicalTag relative="../" tag="checkpoint" text="Checkpoint" />, or <TechnicalTag relative="../" tag="validator" text="Validator" />) you will use a Batch Request to create the Batch that is used.

Reviewer: ```suggestion
A Batch Request is always used to build a Batch. For example, when you run a <TechnicalTag relative="../" tag="checkpoint" text="Checkpoint" /> or use a <TechnicalTag relative="../" tag="validator" text="Validator" />), you'll use a Batch Request to create the Batch.
```

## Use cases

> When you create Expectations, you will need to provide a Batch of data for your Expectations to test against.  For this processes, you will therefore need a Batch Request to get the Batch.

Reviewer: ```suggestion
When you create Expectations, you'll need to provide a Batch of data to test your Expectations. To get the Batch of data, you'll use a Batch Request.
```
